### DISCUSSION TITLE
Need help with Home Assistant config

### DISCUSSION THREAD

**Discussion by bxlouis** [htpcBeginner/docker-traefik#138] [Support]
Hello,

I am trying to setup Home Assistant but I have a hard time finding appropriate documentation.
My whole docker stack runs on a Synology DS918+ with Traefik 2.

First I don't understand why in the [docker-compose-t2.yml](https://github.com/htpcBeginner/docker-traefik/blob/master/docker-compose-t2.yml) example, there are 2 different configurations for Home Assistant running at the same time. One "hass-core" running "home-assistant:0.113.3" and one "hassio" running "home-assistant:0.111.0".

Is it really how it is supposed to be setup or is it an error in the config file, as this config is marked as temporary?

**Comment by SimpleHomelab** [comment]
I have updated the docker compose files and added some details. Ignore hassio. That was my old setup. It is still on there because I am migrating from the old hassio on docker setup to home assistant core.
